src/docs/devices/Adonno-Tagreader/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Adonno Tagreader
+date-published: 2022-04-16
+type: sensor
+standard: global
+board: esp8266
+difficulty: 1
+made-for-esphome: false
```

Comment:
This is actually true

Reply:
Oh sorry, I couldn't see anywhere it was officially stated on the page but I'll correct it.